### PR TITLE
Allow to delete last cluster

### LINKED ISSUE #6228: [UI] Allow user to delete all K8s cluster
#### Current Behavior

When user tries to remove last remaining (or only available K8s cluster), user gets the below message and user is not able to delete the cluster.
Screen Shot 2022-09-20 at 11 27 23 AM

#### Desired Behavior

Allow user to delete all connected K8s cluster.

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by MUzairS15**
Signed-off-by: MUzairS15 <muzair.shaikh810@gmail.com>

**Description**

This PR fixes #6228 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
When the last cluster is deleted, again the contexts are added,( we are again fetching from the `kubeconfig`).
Why this behaviour?? // @Revolyssup

**Comment by Chadha93** [comment]
Restarting the failed integration test.

**Comment by leecalcote** [comment]
All green. 👍

File: ui/components/MeshConfigComponent.js [hunk 1]
```diff
@@ -989,8 +989,8 @@ function MesherySettingsNew({ classes, enqueueSnackbar, closeSnackbar, updatePro
           setMeshsyncSubscription({ type : actionTypes.SET_MESHSYNC_SUBSCRIPTION, meshSyncState : newMeshSyncState })
           handleError("MeshSync could not be reached")("MeshSync is unavailable");
         } else {
-          let publishEndpoint = res.controller.status.substring("ENABLED".length)
-          enqueueSnackbar(`MeshSync was successfully pinged. Publishing to ${publishEndpoint} `, {
+          let publishEndpoint = res.controller.status.substring("ENABLED".length).trim()
+          enqueueSnackbar(`MeshSync was successfully pinged. ${publishEndpoint != "" ? `Publishing to ${publishEndpoint}` : ""}`, {
```
Comment (leecalcote) [inline ui/components/MeshConfigComponent.js:993]: @MUzairS15 there is an opportunity here to say that MeshSync is NOT publishing to Meshery Broker and to show this as a Warning (yellow) snackbar instead. Is this feasible?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Pull request merged**